lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2560,6 +2573,11 @@ createAcceleratorReceive(SILBuilder &B, SILLocation loc, SILType valueTy,
   return B.createBuiltin(loc, name, valueTy, subMap, operands);
 }
 
+static SILValue getSingleValueResult(GraphOperationInst *inst) {
```

Comment:
Isn't this simple enough to be inlined?

Reply:
i like having this helper function -- in most cases, an graph op inst returns a single result, so this util function is called for convenience and also clarity. when we use the builtin tfop, since `BuiltinInst` is `SingleValueInstruction`, we naturally get this kind of API (though processing a multi-result tfop would require extra code of getting a tuple-typed result and unpack the tuple).